libs/ui-toolkit/src/components/theme-switcher/theme-switcher.tsx

```diff
@@ -14,4 +21,31 @@ export const ThemeSwitcher = ({ className }: { className?: string }) => {
       {theme === 'light' && <MoonIcon />}
     </button>
   );
+  const toggles = [
+    {
+      value: 'dark',
+      label: t('Dark mode'),
+    },
+    {
+      value: 'light',
+      label: t('Light mode'),
+    },
+  ];
+  return withMobile ? (
+    <>
+      <div className="flex grow gap-6 md:hidden whitespace-nowrap">
+        {button}{' '}
+        <Toggle
```

Comment:
chore (blocking): can we align this to the right (we can add the small on/off toggle later)

![Screenshot 2023-01-09 at 23 02 22](https://user-images.githubusercontent.com/6803987/211483510-497b9d55-3873-4801-8879-249b365c6c4c.jpg)

Reply:
I aligned a toogle to the right. I decided to use such one from our toolkit instead a switch we had in design. I think we should keep one pattern of used form elements